spec/mailers/user_mailer_spec.rb

```diff
@@ -608,6 +608,28 @@
         expect(email).to have_text_part_content(report.url)
       end
     end
+
+    describe "translation" do
+      before { Timecop.freeze("2022-03-14 13:27:09 +0000") }
```

Comment:
Can you use Rails' [time helpers](https://api.rubyonrails.org/v6.0/classes/ActiveSupport/Testing/TimeHelpers.html) instead?